### PR TITLE
Object method files and linters configuration

### PR DESCRIPTION
**Pull request by sserwangaedrisa**
Object method files and linters configuration.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const addButton = document.getElementById('AddButton');
+const booksSection = document.getElementById('Books-section');
+
+const keys = Object.keys(localStorage);
+if (localStorage.length > 0) {
+  keys.forEach((key) => {
+    const newTitle = document.createElement('p');
+    newTitle.textContent = JSON.parse(localStorage.getItem(key)).Title;
+    const newAuthor = document.createElement('p');
+    newAuthor.textContent = JSON.parse(localStorage.getItem(key)).Author;
```
Comment (arnoldnekemiah) [inline index.js]: I see you have tried a path to using `LocalStorage`, but the approach is not viable and You are not accessing Local Storage
image
From the Image above you can see that data is not being loaded into local Storage.
Internalize the approach below and work on the remaining;

 _check if the local storage is empty, then add empty array_
```
if (localStorage.getItem('Added books') == null) {
  localStorage.setItem('Added books', JSON.stringify([]));
}
```
That is an example on how to handle localStorage, that way you can make functions referencing it by key

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+.vscode/
```
Comment (arnoldnekemiah) [inline .gitignore:2]: What is this. Is it necessary?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.classList.add('Remove');
+    removeBtn.setAttribute('id', title.value);
+
+    removeBtn.addEventListener('click', () => {
+      localStorage.removeItem(removeBtn.id);
+      window.location.reload();
+    });
+
```
Comment (arnoldnekemiah) [inline index.js]: Create a function for removing button as suggested i the guidelines

File: index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    booksSection.appendChild(newTitle);
+    booksSection.appendChild(newAuthor);
+    booksSection.appendChild(removeBtn);
+    booksSection.appendChild(horizontalLine);
+  }
+  title.value = '';
+  author.value = '';
```
Comment (arnoldnekemiah) [inline index.js]: You can also write a function to clear inputs after submission.

File: index.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    booksSection.appendChild(horizontalLine);
+  });
+}
+
+addButton.addEventListener('click', () => {
+  if (title.value.length !== 0) {
+    const inputs = {
+      Title: title.value,
+      Author: author.value,
+    };
```
Comment (arnoldnekemiah) [inline index.js]: This should also be a function, such that you can invoke it whenever you need it.

**Review by arnoldnekemiah — CHANGES_REQUESTED** [review]
Hello @sserwangaedrisa ,
I see you have tried to do the work as best as you understood the assignment. But there are Changes required to make your project functional;

- Make sure you follow the guide lines of the project.

**Comment by sserwangaedrisa** [comment]
I'm going to work on it tomorrow..

**Comment by sserwangaedrisa** [comment]
Hey there, this is how i tried to modify the project

https://github.com/sserwangaedrisa/Awesome-Books-Store/pull/2

**Review by arnoldnekemiah — APPROVED** [review]
Congs @sserwangaedrisa , 
Your Project is complete, There is nothing more to say other than, it's time to merge it.
well-done

I will inform you of the next level of the project. For now Merge the branch

**Comment by arnoldnekemiah** [comment]
Make sure Your linters are running I see no history of linter checks in the Pull request

**Review by arnoldnekemiah — CHANGES_REQUESTED** [review]
Please make sure linters are well configured. Your project is missing `linters.yml` file in the _.github/workflow_s folder.
That is why there are no checks

**Pull request merged**